### ISSUE TITLE
Censoring

### ISSUE THREAD

**Issue by trident-systems-fonz** [trident-systems-fonz/fonz-issues#20]
For summary Activities>Lining>Bottom>Bluenose>North-east: Catches (A.05.01.03.01.catchYear) ...

Looks like all recent data are censored for catch and catch rate (in this and other activity segments)

**Comment by nokome** [comment]
Yes, there is an issue here. All terminal nodes in the activities tree have censoring for the last 5 years. Non-terminal nodes seem to be OK.

**Comment by nokome** [comment]
Issue arose because the flag for missing values in input data files was changed from "NULL" to "[NULL]". This has been dealt with in `catcheffort_load.py`
